hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/tenant/OMTenantCreateRequest.java

```diff
@@ -350,9 +350,9 @@ public OMClientResponse validateAndUpdateCache(
           omVolumeArgs, volumeList, omDBTenantState);
 
     } catch (IOException ex) {
+      exception = ex;
```

Comment:
@symious , this one can be reverted too.

Reply:
Sure, updated.